util/config.txt

```diff
@@ -0,0 +1,5 @@
+config1:abc
+
+config2:123
```

Comment:
This config value should be 567
123 was the old value

Reply:
added correct value

Reply:
@ademotest see the latest commit for the change